### PR TITLE
Switched from colorize to ansi.

### LINKED ISSUE #5: Consider moving from colorize to ansi
Hi there! 

Fantastic tool – thanks so much for building this! The integration with middleman-s3_sync is _very_ helpful. 

Opening this issue because I saw that this gem depends on colorize, which is [licensed under the GPL](https://github.com/fazibear/colorize#license), not a permissive license like on this project. [middleman-s3_sync had the same issue](https://github.com/fredjean/middleman-s3_sync/issues/88#issuecomment-136012660) and switched to [ansi](https://github.com/rubyworks/ansi). If that sounds cool to you, I'm happy to switch over this awesome tool.

### PR DESCRIPTION
**Pull request by ashfurrow**
Not sure where the ansi mixin for String normally goes – happy to move it.

Fixes #5.

### CODE REVIEW

**Comment by leighmcculloch** [comment]
I'm not sure there's a usual place, but how about putting it in `/lib/middleman-cdn/mixins/string.rb`. 

I'd also be welcome to not using the mixin and instead using `ANSI.red{ "Hi" }` so that the gem no longer alters standard types, but the former would definitely suffice.

**Comment by ashfurrow** [comment]
Sounds good, I'll move it over to the code style.

**Comment by ashfurrow** [comment]
Sorry for the delay on this, I'll get it done this weekend.

**Comment by ashfurrow** [comment]
OK, this is updated. A few notes:

- I had to change [this line](https://github.com/leighmcculloch/middleman-cdn/pull/7/files#diff-e97162f97297fe3206c24a1b07bca7cfR75). I wasn't sure what the Ruby convention is – `try` wasn't working for me in pry – so I added a test for what I _think_ it should do. 
- Red and bold text was different [here](https://github.com/leighmcculloch/middleman-cdn/pull/7/files#diff-ff0dae7a2a31e4c54bbc8d32127f0aa3R44). This could be a nested block, as well. Dunno what you prefer. 

Let me know what you think :cake:

File: spec/lib/middleman-cdn/commands_spec.rb [hunk 1]
```diff
@@ -15,6 +15,12 @@
         described_class.say_status(nil, "a status")
       end
     end
+
+    context "with a cdn" do 
+      it "should say" do
+        described_class.say_status("the cdn", "a status")
+      end
+    end
```
Comment (leighmcculloch) [inline spec/lib/middleman-cdn/commands_spec.rb]: This test doesn't really test much, neither does the test above it that I wrote either. Sorry about that. Leave it as it is at the moment, I'll try and improve on them as a whole post merge.

File: lib/middleman-cdn/commands.rb [hunk 2]
```diff
@@ -72,7 +72,7 @@ def cdn_invalidate(*args)
 
       def self.say_status(cdn, status, newline: true, header: true, wait_enter: false)
         message = ""
-        message << "#{:cdn.to_s.rjust(12).green}  #{cdn.try(:yellow)}" if header
+        message << "#{ANSI.green { :cdn.to_s.rjust(12)} }  #{ANSI.yellow{ cdn } unless cdn.nil? }" if header
```
Comment (leighmcculloch) [inline lib/middleman-cdn/commands.rb]: There's unevent whitespace on the closing `}`s of the string interpolations. I can fix that post merge though, no big deal.

**Comment by leighmcculloch** [comment]
Thanks, this looks great!

**Comment by ashfurrow** [comment]
Super duper, thanks for the helpful feedback!

**Comment by leighmcculloch** [comment]
This has been pushed to rubygems in v0.1.14.

**Pull request merged**